From 5d33b53c9330c3687ac11cc749e81fc0dae4ca7f Mon Sep 17 00:00:00 2001
From: Tulio Magno Quites Machado Filho <tuliom@redhat.com>
Date: Mon, 20 Feb 2023 14:57:29 -0300
Subject: [PATCH] Changes the path to gtest

---
 flang/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flang/CMakeLists.txt b/flang/CMakeLists.txt
index b048ea220e20..03bbfd9b8ec0 100644
--- a/flang/CMakeLists.txt
+++ b/flang/CMakeLists.txt
@@ -171,7 +171,7 @@ if (FLANG_STANDALONE_BUILD)
 #Handle unittests when out-of-tree
   set(FLANG_GTEST_AVAIL 0)
   if (FLANG_INCLUDE_TESTS)
-    set(UNITTEST_DIR ${LLVM_THIRD_PARTY_DIR}/unittest)
+    set(UNITTEST_DIR ${LLVM_MAIN_SRC_DIR}/utils/unittest)
     if(EXISTS ${UNITTEST_DIR}/googletest/include/gtest/gtest.h)
       if (NOT TARGET llvm_gtest)
         add_subdirectory(${UNITTEST_DIR} third-party/unittest)
-- 
2.39.1

