From 1f00502316d003f36a87ab25f51e1fc2d3274a5b Mon Sep 17 00:00:00 2001
From: Tom Stellard <tstellar@redhat.com>
Date: Fri, 19 Feb 2021 19:46:12 -0800
Subject: [PATCH 2/2] [PATCH][flang] Fix build with gcc 11

---
 flang/runtime/unit.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/flang/runtime/unit.cpp b/flang/runtime/unit.cpp
index 85d83ec..dbd4b6b 100644
--- a/flang/runtime/unit.cpp
+++ b/flang/runtime/unit.cpp
@@ -12,6 +12,7 @@
 #include "lock.h"
 #include "unit-map.h"
 #include <cstdio>
+#include <limits>
 #include <utility>
 
 namespace Fortran::runtime::io {
-- 
1.8.3.1

