diff -Naur flang-11.0.0rc3.src.orig/docs/conf.py flang-11.0.0rc3.src/docs/conf.py
--- flang-11.0.0rc3.src.orig/docs/conf.py	2020-09-25 10:09:59.043417169 +0000
+++ flang-11.0.0rc3.src/docs/conf.py	2020-09-25 10:11:00.715930278 +0000
@@ -46,7 +46,7 @@
   else:
     source_parsers = {'.md': 'recommonmark.parser.CommonMarkParser'}
   source_suffix['.md'] = 'markdown'
-  extensions.append('sphinx_markdown_tables')
+  #extensions.append('sphinx_markdown_tables')
 
   # Setup AutoStructify for inline .rst toctrees in index.md
   from recommonmark.transform import AutoStructify
